lib/beaker/host_prebuilt_steps.rb

```diff
@@ -367,13 +367,15 @@ def copy_ssh_to_root host, opts
     def hack_etc_hosts hosts, opts
       etc_hosts = "127.0.0.1\tlocalhost localhost.localdomain\n"
       hosts.each do |host|
-        ip = host['vm_ip'] || host['ip'].to_s
-        hostname = host[:vmhostname] || host.name
-        domain = get_domain_name(host)
-        etc_hosts += "#{ip}\t#{hostname}.#{domain} #{hostname}\n"
+        unless host['platform'] =~ /windows/
```

Comment:
@james-powis can you provide some context on why this is restricted to non-windows platforms? I'm pretty sure that internally here at puppet, we will still want this functionality on windows as well as unix based systems.